doc/index.rst

```diff
@@ -45,7 +50,7 @@ Check out these demos to see the PennyLane-Forest plugin in action:
     <div class="row">
 
 .. title-card::
```

Comment:
I think this `title-card` and the next one are not rendered properly (see [link](https://xanaduai-pennylane--107.com.readthedocs.build/projects/forest/en/107/index.html)).

Reply:
Looks like the `index-card` directive no longer supports figures. None of the [available directives in the xanadu-sphinx-theme](https://github.com/XanaduAI/xanadu-sphinx-theme#directives) are a perfect match for what I would imagine was being rendered before. There is a reference to a "Related Demo" directive, but it doesn't look like that's been finished yet. What would you recommend?

Reply:
I would just recommend removing the figure. But maybe @josh146 has a better solution.

Reply:
I went ahead with that for now (and added a description). Happy to change it later if Josh has a suggestion!